docs/source/components/nodes/imu.rst

```diff
@@ -79,6 +79,20 @@ Usage
     // useful to reduce device's CPU load  and number of lost packets, if CPU load is high on device side due to multiple nodes
     imu->setMaxBatchReports(10);
 
+IMU devices
+###########
+
+List of devices that have an IMU sensor on-board:
```

Comment:
@MOj0, would you mind updating product names to align with the naming convention (IoT, PoE, Lite, and Pro), please? And could you also add links to hardware docs, please?
cc. @Erol444 
Thanks!

Reply:
Opened new PR [here](https://github.com/luxonis/depthai-python/pull/360)